### PR TITLE
[fix] withdraw all and total deposited value calculation

### PR DESCRIPTION
**Pull request by garguelles**
## Checklist (Mandatory):

- [x] I have named the PR correctly
- [x] I have tested new code locally
- [x] I have done a smoke test of the larger feature locally
- [x] I have added at least one reviewer
- [x] I have assigned the PR to myself
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts
- [x] I have ensured any new UI changes comply with UX/UI designers 

## Checklist (Optional):

- [x] I have added necessary documentation, including instructions how to test
- [x] I have written the code according to the adopted style guide and linter doesn't fail
- [x] I have confirmed no errors were introduced in the developer console with the PR

## Additional notes

```
What does this implement/fix? Explain your changes.
- use withdraw all if withdrawValue percent === 100
- use getShareInUSD instead of token price
- fix formatting
Are there any scripts that need running (eg. yarn install)?
Are there any edge cases that need checking?
Are there any specific test instructions?
Are there any Loom videos or screenshot that can be linked explaining the changes better?
```

### CODE REVIEW

**Review by pajicf — CHANGES_REQUESTED** [review]
This kinda just covers the bug under the rug, I think the problem lays in the calculation itself, more precisely the `getShareInFormattedToken` function receives correct values for shares and pricePerShares but the output result is not entirely correct.

Here's a proposition on how to solve and correct me if this brakes something somewhere else.
I think it should be safe to assume `getShareInFormattedToken` will always receive BigNumber, hence we can use BigNumber again for multiplication and not loose on precision there.

The second part of the problem is rounding of `formattedUserShares` inside withdrawForm, where it rounds up and makes an incorrect number in return. This should be fixable by providing the `BigDecimal.RoundingModes.DOWN` mode as parameter of the rounding function

File: src/components/molecules/WithdrawForm/withdrawForm.molecule.tsx [hunk 1]
```diff
@@ -171,7 +171,7 @@ const WithdrawForm: React.FC<IVaultForm> = (props) => {
             className={styles.actionButton}
             disabled={isWithdrawSomeAssetsDisabled || isDepositingAssets || isWithdrawingAllAssets}
             loading={isWithdrawingAssets}
-            onClick={withdrawAssets(false)}
+            onClick={withdrawAssets(withdrawValue.percent === 100)}
```
[the remark this answers is not in the document]
Comment (garguelles) [reply src/components/molecules/WithdrawForm/withdrawForm.molecule.tsx:174]: > This kinda just covers the bug under the rug, I think the problem lays in the calculation itself, more precisely the `getShareInFormattedToken` function receives correct values for shares and pricePerShares but the output result is not entirely correct.
> 
> Here's a proposition on how to solve and correct me if this brakes something somewhere else. I think it should be safe to assume `getShareInFormattedToken` will always receive BigNumber, hence we can use BigNumber again for multiplication and not loose on precision there.
> 
> The second part of the problem is rounding of `formattedUserShares` inside withdrawForm, where it rounds up and makes an incorrect number in return. This should be fixable by providing the `BigDecimal.RoundingModes.DOWN` mode as parameter of the rounding function

Will try this. Thanks

**Review by pajicf — APPROVED** [review]
🚀
